### ISSUE TITLE
Xinyan's C6 buffs only the charged attack, not her whole damage.

### ISSUE THREAD

**Issue by RavenousBastard** [JenShin-368/GenshinImpactCalculator#91]
On GI Damage Calculator, when activating Xinyan's C6, all her damage is being buffed based from 50% of her DEF. 

Only the charged attack should receive the buff from 50% of her DEF, as per the C6 description.

Thanks for the awesome site, it has been really helpful! Least I can do is help make it better.

**Comment by Albedo-0** [comment]
yes, but no, let me explain, in sixth constellation she gains an ATK bonus equal to 50% of her DEF, only when she is doing a charge attack she gains that boost, but what happen when she produce  adicional damage?, for example, the prototype archaic or the debate club. She starts a charge attack, gain that boost, and then the passive of these weapons activates, they uses the ATK(with the bonus) that Xinyan has in that moment. So you have to be careful when you change things in the app

**Comment by RavenousBastard** [comment]
Wait... so her constellation is not written correctly?
